### ISSUE TITLE
Hey Wiki settings don't work

### ISSUE THREAD

**Issue by egogowa** [mc-wiki/minecraft-mod-heywiki#69]
### Steps to reproduce

1. Open the game. 2. Go to the mod menu. 3. Press the Hey Wiki settings option button. 4. The option button does not work

### Describe the bug

Option button does not work, giving the following error: Java.lang.IllegalStateException:Unexpected value: Automatic

### Expected behavior

Option button works as expected

### Mod version

1.5.7

### Game version

1.21

### Operating system

Windows 10

### Mod loader

Fabric

### Mod loader version

0.15.11-1.21

### How frustrating is this bug?

5

### Additional information and screenshots

Other mods used: 
sodium-fabric-0.5.11, placeholder-api-2.4.1, modmenu-11.0.1, lithium-fabric-mc1.21-0.12.7, cloth-config-15.0.128-fabric, architectury-13.0.6-fabric

Screenshot:
(external text in Russian)
![2024-08-20_101628](https://github.com/user-attachments/assets/2fb8aabd-f890-46f6-b6ef-da89d265e2d8)

**Comment by Dianliang233** [comment]
nice

**Comment by Dianliang233** [comment]
@egogowa Please try to delete your `.minecraft/config/heywiki.json`, start the game, change a few settings and let me know if it works for you.

**Comment by egogowa** [comment]
Yeah, it worked, but unfortunately only once. After game restart the options button is inactive again, however, all previously changed settings are saved
I have to say that the options were active the first time I ran with the mod, but not the next. But I didn't change anything right away

**Comment by Markus-Rost** [comment]
Can you show the content of your `.minecraft/config/heywiki.json` when it's not working?

**Comment by egogowa** [comment]
[heywiki.json](https://github.com/user-attachments/files/16676921/heywiki.json)

**Comment by egogowa** [comment]
meh

{
  "requiresConfirmation": false,
  "language": "en",
  "zhVariant": "Автоматически",
  "raycastReach": 5.2,
  "raycastAllowFluid": false
}

**Comment by Markus-Rost** [comment]
1.6.1 should have fixed that issue, can you try with the updated version of the mod?

**Comment by egogowa** [comment]
I installed this as soon as I noticed it during testing. Zero effect 😅. The sent json already relates to 1.6.1

**Comment by egogowa** [comment]
Yeah, I didn't think to delete the old json right away. Deleted it and it worked. Thanks a lot!

**Comment by Dianliang233** [comment]
seems to be fixed, please open another issue if you encounter any more problems. Thanks!
